packml_smach/src/executeState.py

```diff
@@ -33,7 +37,13 @@ def __init__(self):
         
     def execute(self, userdata):
         rospy.loginfo('Executing state Execute')
-        if userdata.execute_in == True:
+
+        # Receive Stop Tag
+        var4 = config.client.get_node("ns=3;s=\"PackML_Status\".\"UN\".\"Cmd_Stop\"")  
+        var4.set_value(True)
+        config.stopTag = var4.get_value()
+        print("Received stop command: ", config.stopTag)
```

Comment:
What exactly did you receive.  It appears that you created `var4`, set it to `True`, and then read value of `var4` into `config.stopTag`, which you check below.